components/notability/commons/notability_checker.lua

```diff
@@ -102,8 +104,10 @@ end
 
 function NotabilityChecker._calculatePlayerNotability(player)
 	player = mw.ext.TeamLiquidIntegration.resolve_redirect(player)
-	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]'
-	for i = 2, 10 do
+
+	local conditions = '[[players_p' .. tostring(1) .. '::' .. player .. ']]' ..
+		' OR [[participant::' .. player .. ']]'
+	for i = 2, 20 do
```

Comment:
broodwar needs up to 40, sc2 up to 30, some wikis seem to have no limit (limit 10 as default, but overwritable)

Reply:
Done, now available in config